src/Components/PokeCard.tsx

```diff
@@ -1,17 +1,16 @@
-import React from 'react'
-import '../CSS/PokeCard.css'
-import PokeDetails from './PokeDetails.tsx'
-
+import React from 'react';
+import '../CSS/PokeCard.css';
+import { Link } from 'react-router-dom';
 
 //do we need type also?
 
 const PokeCard = ({name, sprite, id}) => {
   
  return(
-   <div>
+   <Link to={`${id}`} className='poke-card' >
      <img src={sprite} alt='pokemon character'/>
```

Comment:
You can leave this alt blank `alt=""` since you are displaying the name on the card. Also, make sure to use an `<h2>` for the name, `<h1>` should only be used once at the top of the page.